cmd/virt-tail/main.go

```diff
@@ -217,7 +222,7 @@ func main() {
 	// wait for all errgroup goroutines
 	if err := g.Wait(); err != nil {
 		// Exit cleanly on clean termination errors
-		if !(errors.Is(err, context.Canceled) || errors.Is(err, &TermFileError{})) {
+		if !(errors.Is(err, context.Canceled) || errors.Is(err, &TermFileError{}) || errors.Is(err, &SocketFileError{})) {
```

Comment:
Exit code 1 is usually an error, shouldn't you use 0 to terminate gracefully?

Reply:
This is now excluding that kind of error from the branch where we should exit with 1.
In all the other cases (so now including that specific error, it will exit with 0).

Reply:
Ah didn't see the negation in front of the condition, I think the comment misled me, can you move it? It is a bit confusing places right before that if statement.

Reply:
Yeah, I also had to read this line twice